### PR TITLE
WIP: Add org.gnome.gitlab.sophie_h.PikaBackup

### PR DESCRIPTION
**Pull request by sophie-h**
Only WIP because the App Id isn't decided yet. Review welcome.

### CODE REVIEW

**Comment by sophie-h** [comment]
bot, build org.gnome.gitlab.sophie_h.PikaBackup

**Comment by sophie-h** [comment]
bot, build org.gnome.gitlab.sophie_h.PikaBackup

**Comment by sophie-h** [comment]
bot, build org.gnome.gitlab.sophie_h.PikaBackup

**Comment by sophie-h** [comment]
bot, build org.gnome.gitlab.sophie_h.PikaBackup

File: org.gnome.gitlab.sophie_h.PikaBackup.yml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  append-path: /usr/lib/sdk/rust-stable/bin
+
+finish-args:
+  - --filesystem=host
+  - --share=network
+  - --socket=wayland
+  # X11
+  - --share=ipc
+  - --socket=x11
+  - --socket=fallback-x11
```
Comment (nedrichards) [inline org.gnome.gitlab.sophie_h.PikaBackup.yml:19]: you need one of socket=x11 or fallback-x11, not both. fallback-x11 is preferred and means your app will use wayland by default in a wayland environment.

Comment (sophie-h) [reply org.gnome.gitlab.sophie_h.PikaBackup.yml:18]: Left x11 in to keep compatibility with debian old-stable (flatpak < 1.0), but probably I'm missing a million required features there anyways. Removed it.

Comment (sophie-h) [reply org.gnome.gitlab.sophie_h.PikaBackup.yml:18]: And thanks for the feedback!

Comment (nedrichards) [reply org.gnome.gitlab.sophie_h.PikaBackup.yml:18]: < Flatpak 1.2 is about 20% of Flathub downloads fwiw https://ahayzen.com/direct/flathub.html but yeah, that's fair! If you absolutely know you need a feature that isn't present in an older version you can use --require-version like so https://github.com/flathub/com.visualstudio.code/blob/master/com.visualstudio.code.yaml#L11

**Comment by nedrichards** [comment]
bot, build org.gnome.gitlab.sophie_h.PikaBackup

**Comment by nedrichards** [comment]
@sophie-h you can use that command as well in these PRs to run a new test build any time you make a change.

**Comment by bilelmoussaoui** [comment]
> Only WIP because the App Id isn't decided yet. Review welcome.

Feel free to request to add your application to GNOME Circle (applications made for GNOME) https://gitlab.gnome.org/Teams/Circle/-/issues
this should give you access to a bunch of nice stuff, along with the possibility to use some nice app-id as well, not sure about the details though.

**Comment by sophie-h** [comment]
> Feel free to request to add your application to GNOME Circle

Wasn't sure if my app is too new. But I will try if this issues doesn't show any progress.

https://gitlab.gnome.org/Infrastructure/Infrastructure/-/issues/398

**Pull request closed without merging**